docker/prod/Dockerfile-nginx

```diff
@@ -1,10 +1,6 @@
 FROM xcgd/nginx-vts:stable
 
-RUN chown -R nginx:nginx /etc/nginx && \
-    chown nginx:nginx /var/cache/nginx && \
-    chown -R nginx:nginx /var/log/nginx && \
-    mkdir /var/run/nginx && \
-    chown nginx:nginx /var/run/nginx
+RUN mkdir /run/nginx && chown -R 65534:65534 /run /var/cache /etc/nginx
```

Comment:
Does it make a difference in this case if the user owns `/var/cache` or /`var/cache/nginx/`?

Reply:
Generally effect is similar from nginx process point of view, It's just easier to debug when you enter container manually.